report/4.Solution/GeneratedCppCode.tex

```diff
@@ -0,0 +1,86 @@
+% !TeX root = ../main.tex
+
+\subsection{Generated C++ code}
+While the previous section is primarily focused on the code generation from an abstract point of view, this section focuses on the generated C++ code.
+Since C++ is the output language of the HCL compiler, all the language concepts of HCL have to be represented in valid C++ code.
+
+\textbf{Primitive types}
+The primitive types of HCL includes numbers, text and booleans. 
+Booleans are straightforward, as HCL's boolean behaves exactly like the C++ boolean type.
+This means that the regular C++ boolean type can be used as the output type of HCL booleans.
+
+As HCL aims for simplicity, only one number type is introduced. 
+This type needs to be able to store both integers and non-integers. 
+The simple solution is to turn all HCL numbers into C++'s double type.
```

Comment:
Change "to turn all" -> "to format all"

Reply:
Maybe turn is not the correct word, but i don't think format is correct either (the formating of the number remains the same, however the type is "turned" or changed to the double type). Any alternative suggestions - maybe "change all HCL number types into C++ doubles"? Althoug I still believe that turn can work here.

Reply:
Convert, translate. Tho i think turn is fine

Reply:
Convert is perfect, tywm